data/remnant missions.txt

```diff
@@ -153,20 +221,18 @@ mission "Remnant: Defense 2"
 			names "korath"
 			cargo 3
 			variant
-				"Korath World-Ship B"
-		dialog "You have destroyed the Korath ship that was left over from the raid on <planet>. You can now return there to collect your payment."
+				"Korath Chaser" 5
+		dialog "You have destroyed the Korath ships that were left over from the raid on <planet>. You can now return there to collect your payment."
 	on complete
 		payment 250000
 		dialog `The same woman meets you when you land on <planet>. "Again, thank you," she says. "I will suggest to others that they might offer you similar bounty hunting jobs in the future." She pays you <payment>.`
 		log "Factions" "Remnant" `Recently, Korath ships have begun raiding Remnant worlds. The Remnant have decided not to try to discourage these raids, because they are a perfect opportunity to steal jump drives and other valuable alien technology from the Korath.`
 
-
-
-mission "Remnant: Bounty"
+mission "Remnant: Hunter Bounty"
```

Comment:
The main point to this was to enable a bit of a gradual entry to the Remnant missions, for a number of reasons:
1) It is possible to reach the Remnant flying something as weak as a scoutship, but a mission requiring the player to defeat raiders and world ships would force the player to abandon any hope of progressing in these missions until they can come back with something more along the lines of an SB. That might be a mere hour or two delay for experienced players, but for many that could be quite a bit longer - perhaps long enough for them to forget about the Remnant entirely. 

2) It'd be kind of weird to get sent to track down a raider threatening their ships, and if the player is forced to leave the area to get a better ship and come back months (or maybe even a year or two later having built up to a SB or something from the Wanderer campaign) and find that the Remnant *still* haven't dealt with this obvious threat.

3) ES is primarily about the space fights, and in the Remnant corner of the galaxy there's really only three kinds of enemy ships to fight: The Chasers, the Raiders, and the World-Ship. Jumping straight to dealing with raiders and world ships right off the bat kind of eliminates any possibility of progression, as it just becomes increasing numbers of those three things. Using Chasers for the initial threats (and the initial bounty mission) means the player has a shot at doing them flying a wider variety of ships including many of the light warships from regular human space.

4) The Remnant have a perfectly good light warship that there's really no point to using. If the player doesn't get access to it until after they've already got something capable of defeating raiders and world-ships, unlocking access to a light warship that is probably significantly weaker than whatever they are flying isn't really much of a perk. It means that Starling will always be just a decorative element to diversify Remnant fleets. Much of the time the player is likely to get the cloaking device via the FW campaign, so even having built in cloak isn't that big a benefit. (Unless the player goes the Checkmate route and uses the Starlings cloaking to stealthily dceliver nukes, anyway). 

By changing the warships down to fighters, the player can complete the missions flying regular human warships, possibly even light warships or heavily modified freighters. And without a fleet, too. Then when they unlock the Remnant ships and outfits, the Starling will actually be a potentially worthwhile upgrade for them. It also will mean that the player gets to move up from dealing with Korath chasers to bigger threats, giving them an increasing sense of accomplishment.

Reply:
1. Just because it's possible for someone to show up in a light warship or the like doesn't mean it's likely. Yes, leaving and returning with a heavier ship is easy for an experienced player to do, but it's also only an experienced player that is likely to find their way to Remnant space so early on in the first place. I understand the idea of ramping up the difficulty, but this is content outside of human space and should therefore already start somewhat difficult as it does already.
2. *waves hand*. This can happen anywhere in the game and therefore shouldn't really be worried about.
3. See one.
4. If much of the time the player is already going to have beaten the FW campaign, then see one. As long as cloaks are limited, the Starling will always be a ship of interest to use. 

Warships to fighters is a HUGE drop in difficulty, so if anything I think we have an argument for expanding the Exile arsenal ever so slightly to include a medium warship of some sort, meaning a player could conceivably enter Remnant space with a medium warship, finish the missions, and be able to acquire the Starling while not being too far ahead of it in power.

Reply:
1. True enough, reaching the Remnant in a scout is unlikely, and a light warship probably only a little bit more so. I'm not sure how many players are explorers like me. Once I get something with decent range (like a scoutship loaded down with afterburners, fuel pods, and ramscoops), I like to map out the galaxy and track down all the little corners. Players like that might run into them fairly early. 

Anyway, you make a good case that the challenge for these missions should be higher than a handful of fighters, and I agree. Speaking of which, I would love to have a Korath light and/or medium warship to include here.

Would a larger fleet of fighters be acceptable?

Reply:
gotta say i'm on derpu's side, one needs to be *very* determined to get to the remnant while still flying a light warship.
would it be possible to to spawn a damaged raider with reduced hull points and partially disabled weapons instead?

Reply:
That's true, I could do that.

I wonder how hard it'd be to make a battle-scarred ship... I'll ask on the discord.

Reply:
>Would a larger fleet of fighters be acceptable?

I think that at this point, these missions should be reverted to what they were, and if later on we develop a medium warship for the Exiles then we can swap that in, but I'd rather not have these missions be held back on just that.

Reply:
Would you consider a damaged Raider then? I have created a "Korath Raider (Damaged)" that has about half the shields and hull of a regular raider, and damaged in a variety of ways (a bit slower, has 2 fewer grab strikes, less fuel, no fighters, etc). 

I have swapped out the Chasers in both Remnant Defense 2 and Remnant Bounty for a damaged Raider. Which also means that the Remnant Clean-up Bounty is now actually a unique mission since it faces 4 chasers, which are a different opponent than the Remnant Bounty. (which also fits, since the Clean-up Bounty pays less than the Remnant Bounty).

Reply:
See my response about the Chasers; a "damaged' Raider could conceivably have more drag to be slower and be missing outfits that are considered too "damaged" to work (most importantly the system core), but reducing the shields and hull doesn't make sense in my eyes since the player can't actually *see* that the hull and shield have been reduced. If we had some way of spawning NPC ships so that they showed as being damaged but were still functional, then that would make sense.